### PR TITLE
[Bug] Unable to modify the wrapper classes of upload field

### PR DESCRIPTION
**Pull request by promatik**
This is a fix for #3593.

Upload field was not considering `wrapperAttributes` as an option for the `wrapper` as other fields do.
This PR fixes that, and I also checked every field and fixed some issues similar to this one other fields;

- address_algolia
- browse_multiple
- hidden
- radio
- upload
- upload_multiple

All this fields were not consistent with the standard of `$field['wrapper'] = $field['wrapper'] ?? $field['wrapperAttributes'] ?? [];`

### CODE REVIEW

File: src/resources/views/crud/fields/upload.blade.php [hunk 1]
```diff
@@ -1,11 +1,7 @@
 @php
-   if (!isset($field['wrapper']) || !isset($field['wrapper']['data-init-function'])){
-        $field['wrapper']['data-init-function'] = 'bpFieldInitUploadElement';
-    }
-
-    if (!isset($field['wrapper']) || !isset($field['wrapper']['data-field-name'])) {
-        $field['wrapper']['data-field-name'] = $field['name'];
-    }
+    $field['wrapper'] = $field['wrapper'] ?? $field['wrapperAttributes'] ?? [];
+    $field['wrapper']['data-init-function'] = 'bpFieldInitUploadElement';
```
Comment (pxpm) [inline src/resources/views/crud/fields/upload.blade.php:3]: I think to make this non-BC we need here `$field['wrapper']['data-init-function'] = $field['wrapper']['data-init-function'] ?? 'bpFieldInitUploadElement';`

The same for the `$field['wrapper']['data-field-name']`

I agree that the init function should be ours, but if in code it allowed to be replaced by developer own custom init function, we should continue respecting that, atleast until 4.2.

File: src/resources/views/crud/fields/upload_multiple.blade.php [hunk 2]
```diff
@@ -1,12 +1,7 @@
 @php
-    if (!isset($field['wrapperAttributes']) || !isset($field['wrapperAttributes']['data-init-function'])){
-        $field['wrapperAttributes']['data-init-function'] = 'bpFieldInitUploadMultipleElement';
-    }
-
-    if (!isset($field['wrapperAttributes']) || !isset($field['wrapperAttributes']['data-field-name'])) {
-        $field['wrapperAttributes']['data-field-name'] = $field['name'];
-    }
-
+    $field['wrapper'] = $field['wrapper'] ?? $field['wrapperAttributes'] ?? [];
```
Comment (pxpm) [inline src/resources/views/crud/fields/upload_multiple.blade.php:2]: Apply the same comment as in `upload.blade.php`

**Review by pxpm — CHANGES_REQUESTED** [review]
I think all good, except or the two commented issues.

File: src/resources/views/crud/fields/upload.blade.php (hunk 1 above)
Comment (promatik) [reply src/resources/views/crud/fields/upload.blade.php:3]: @pxpm I agree, I think this was a mistake.
I'll do the changes accordingly your suggestions.

**Comment by promatik** [comment]
@pxpm just applied your suggestion, thanks 👌

**Review by pxpm — APPROVED** [review]
I think this is ok to merge now 👍 

Thanks @promatik

**Pull request merged**